qa/workunits/rados/test.sh

```diff
@@ -36,7 +36,6 @@ for f in \
     api_service api_service_pp \
     api_c_write_operations \
     api_c_read_operations \
-    api_cls_remote_reads \
```

Comment:
I don't see removal of the associated files in the commits. Missed `git add` perhaps?

Reply:
Done